Volume/Views/BrowserView.swift

```diff
@@ -16,15 +16,13 @@ struct BrowserView: View {
     let article: Article
 
     private func incrementShoutouts() {
-        if !userData.articleShoutoutMaxed(article) {
-            userData.incrementShoutoutCounter(article)
-            let currentArticleShoutouts = max(userData.shoutoutsCache[article.id, default: 0], article.shoutouts)
-            userData.shoutoutsCache[article.id, default: 0] = currentArticleShoutouts + 1
-            // swiftlint:disable:next line_length
-            let currentPublicationShoutouts = max(userData.shoutoutsCache[article.publication.id, default: 0], article.publication.shoutouts)
-            userData.shoutoutsCache[article.publication.id, default: 0] = currentPublicationShoutouts + 1
-            Network.shared.apollo.perform(mutation: IncrementShoutoutsMutation(id: article.id))
-        }
+        userData.incrementShoutoutsCounter(article)
+        let currentArticleShoutouts = max(userData.shoutoutsCache[article.id, default: 0], article.shoutouts)
+        userData.shoutoutsCache[article.id, default: 0] = currentArticleShoutouts + 1
```

Comment:
Here you don't need `, default: 0` because you're not reading the cache in this line. Same on line 24.